insonmnia/node/market.go

```diff
@@ -126,6 +129,34 @@ func (m *marketAPI) CancelOrder(ctx context.Context, req *pb.ID) (*pb.Empty, err
 	return &pb.Empty{}, nil
 }
 
+func (m *marketAPI) Purge(ctx context.Context, _ *pb.Empty) (*pb.Empty, error) {
+	orders, err := m.remotes.dwh.GetOrders(ctx, &pb.OrdersRequest{
+		Type:     pb.OrderType_BID,
+		Status:   pb.OrderStatus_ORDER_ACTIVE,
+		AuthorID: pb.NewEthAddress(crypto.PubkeyToAddress(m.remotes.key.PublicKey)),
+		Limit:    dwh.MaxLimit,
```

Comment:
This sometimes leaves some orders if it's total number is greater than `dwh.MaxLimit`. I see no harm here for now.